### PR TITLE
Null buffer handling

### PR DESCRIPTION
**Pull request by tequdev**
Modify the Null Buffer to be handled on the parser side instead of the definition side.

### CODE REVIEW

**Comment by zgrguric** [comment]
@tequdev Feel free to merge this to main.

**Review by zgrguric — APPROVED** [review]
All ok.

File: samples/parser.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ const getByteLength = (state: Field, buffer: Buffer) => {
 const bufferToReadableData = (
   buffer: Buffer,
   state: Field,
-  nullReplaceTo = '0',
+  nullReplaceTo = '00',
```
Comment (zgrguric) [inline samples/parser.ts:49]: Possibly not needed to define default value here.

Comment (tequdev) [reply samples/parser.ts:49]: It is intended to prevent destructive change.

**Pull request merged**